VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -29,20 +31,30 @@ final public class AuthenticationController
     let configuration: AppConfiguration
     let client: VimeoClient
     
+    /// We need to use a separate client to make the actual auth requests, to ensure it's unauthenticated
+    private let authenticatorClient: VimeoClient
+    
     private let accountStore: AccountStore
     
+    /// Set to false to stop the refresh cycle for pin code auth
+    private var continuePinCodeAuthorizationRefreshCycle: Bool = true
+    
     public init(client: VimeoClient)
     {
         self.configuration = client.configuration
         self.client = client
         self.accountStore = AccountStore(configuration: client.configuration)
+        
+        self.authenticatorClient = VimeoClient(appConfiguration: client.configuration)
     }
     
     public init(configuration: AppConfiguration, client: VimeoClient)
```

Comment:
Can you add a comment above this constructor that describes why I would want to pass a configuration other than that used to create the client that I'm passing in?